### DISCUSSION TITLE
Added Premade Scenario for Theaters

### DISCUSSION THREAD

**Discussion by nh905** [microCOVID/microCOVID#996] [Ideas]
Have you considered a premade scenario relating to theatres at 50% capacity?  I did a rough calculation based on 23" wide seats and 38" row spacing, with empty seats to each side and in front to come up with about 15 people within a 7.5 foot radius.  I was not sure what to select for distance - given the seat spacing, I estimate that nearly half the people would be within three feet of me.

Thanks, Norbert

**Comment by blanchardjeremy** [comment]
Tagging @jy2wong as she has been working on scenarios

**Comment by blanchardjeremy** [comment]
Moved this to issue #1010
